### ISSUE TITLE
educationData.RData

### ISSUE THREAD

**Issue by nicolasfstgelais** [Canderson156/eBird-Socioeconomics#19]
I really like the idea of having census data on educational Attainment but it seems that the data is not by county.
Am I wrong?

**Comment by Phytoplanktonic** [comment]
You’re not wrong, but I think it still helps us refine our estimates in a Bayesian frame. The educational attainment data is by race. So the calculation would break down such that we could look at the fraction of say, Asian people in a county, then to that apply the probability of college education given Asian. Example: Maybe there are 10,000 people in the county, 1/10 Asians, and 2/10 of Asians have a college degree. That would help us estimate that there are probably 200 college-educated Asians in that community. Then we can apply the Fish + Wildlife estimates for the probability of being a college-educated (0.28) birder and Asian birder (0.04), which would tell us that we would expect ~2 (2.24) college-educated Asian birders in that community.

Once we have probabilistic estimates of the birder breakdown, we could compare to the Ebird data, to see if regions *projected* to have a higher concentration of certain types of birders show different patterns. (It’s never going to be as good as having the actual data, but the probability-based estimates are better than nothing). :)

**Comment by nicolasfstgelais** [comment]
Ok! sounds pretty cool!
